scripts/state_backup.sh

```
@@ -1,30 +1,53 @@
-  local project=$(${DASHMATE_BIN} config envs --config "$cfg" | awk -F= '$1=="COMPOSE_PROJECT_NAME"{print $2}')
+  local project=$("${DASHMATE_CMD[@]}" config envs --config "$cfg" | awk -F= '$1=="COMPOSE_PROJECT_NAME"{print $2}')
   echo "${project}_${ABCI_VOLUME_SUFFIX:-drive_abci_data}"
 }
+transfer_help() {
+  local component=$1
+  local archive=$2
+  printf 'Move the archive manually, for example:\n  scp %s user@remote:\nThen on the target host run:\n  scripts/state_backup.sh import %s %s [--config <name>]\n' "$archive" "$component" "$(basename "$archive")"
+}
 abci_export_state() {
-  local archive=${1:-drive_abci_state_$(abci_timestamp).tar.gz}
   local cfg=$(abci_resolve_config "${2:-}")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)